certified-connectors/Nodefusion Portal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,166 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Nodefusion Portal",
+    "description": "A Nodefusion Portal Custome Connector is used for various actions, such as opening support case, or subscription activation",
+    "contact": {
+      "name": "Nodefusion",
+      "url": "https://www.nodefusion.com",
+      "email": "nodefusion@nodefusion.com"
+    },
+    "license": {
+      "name": "MIT",
+      "url": "http://opensource.org/licenses/MIT"
+    },
+    "version": "5.1.0.4732"

```

Comment:
@hkusulja This version property belongs to your connector. It should be `"1.0"` or `"1.0.0"` for all the new connectors.

Reply:
updated